File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums):
        
        if len(nums) == 1:
            return 1
        
        cnt = 0
        for i in range(1, len(nums)):
            if nums[i-1] == nums[i]:
                cnt += 1
            else:
                nums[i-cnt] = nums[i]
                
        return len(nums) - cnt

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.c
int removeDuplicates(int* nums, int numsSize) {
	if (numsSize == 1) return 1;
	
	size_t cnt = 0;

	for (int i = 1; i < numsSize; i++) {
		if (nums[i - 1] == nums[i])
			cnt++;
		else
			nums[i - cnt] = nums[i];
	}

	return numsSize - cnt;
}